server/apps/archive_broadcast/broadcast.py

```diff
@@ -293,5 +295,71 @@ def spike_item(self, original):
         If Original item is re-write then it will remove the reference from the broadcast item.
         :param: dict original: original document
         """
-        if original.get('rewrite_of'):
+        broadcast_items = [item for item in self.get_broadcast_items_from_master_story(original)
+                           if item.get(ITEM_STATE) not in PUBLISH_STATES]
+
+        if broadcast_items:
```

Comment:
This check is not necessary, since the following `for` loop will not run if `broadcast_items` ends up being an empty list.